### ISSUE TITLE
slivar tsv: extract QUAL and FORMAT fields

### ISSUE THREAD

**Issue by Madelinehazel** [brentp/slivar#104]
Hello, 
Is there any way to extract the QUAL field for each variant, as well as the alternate allele depths for each sample when using ```slivar tsv```?

Thanks,
Madeline

**Comment by brentp** [comment]
Hi,
you can derive the AD allele depths using DP *AB which are output for each sample.
For qual, you can't get that currently. I'll add it for next release.
-B

**Comment by Madelinehazel** [comment]
Awesome, thanks Brent!

**Comment by Madelinehazel** [comment]
(Any estimates as to a timeline for next release?)

**Comment by brentp** [comment]
If you'll try this attached binary and verify it works for you, I can get a new release out this week. (You'll know you have the right version in your path as it will show v0.2.6 when run without args.)
[slivar.gz](https://github.com/brentp/slivar/files/7276331/slivar.gz)

**Comment by justme66** [comment]
Hi Brent,
Is there any way I can get GQ for each variant? Can I modify the 'src/slivarpkg/tsv.nim' and create binary slivar by myself to get GQ or other info for each variant?

Thanks,
Emma

**Comment by brentp** [comment]
@justme66 , yes, that's possible. you could write a getGQ function copying from `getAB` and `getDP` and then also see the call-site for those functions.

**Comment by justme66** [comment]
Thank you. Brent.
